Sources/AIProxy/OpenAI/OpenAIService.swift

```diff
@@ -186,6 +186,42 @@ public final class OpenAIService {
 
         return try JSONDecoder().decode(OpenAICreateTranscriptionResponseBody.self, from: data)
     }
+    
+    /// Initiates a create text to speech request to v1/audio/speech
+    ///
+    /// - Parameters:
+    ///   - body: The request body to send to aiproxy and openai. See this reference:
+    ///           https://platform.openai.com/docs/api-reference/audio/createSpeech
+    /// - Returns: The audio file content. See this reference:
+    ///            https://platform.openai.com/docs/api-reference/audio/createSpeech
+    public func createTextToSpeechRequest(
+        body: OpenAITextToSpeechRequestBody
+    ) async throws -> Data {
+        let session = AIProxyURLSession.create()
+        let request = try await AIProxyURLRequest.create(
+            partialKey: self.partialKey,
+            serviceURL: self.serviceURL ?? legacyURL,
+            clientID: self.clientID,
+            proxyPath: self.resolvedPath("audio/speech"),
+            body:  try JSONEncoder().encode(body),
```

Comment:
You can use a helper here: `try body.serialize()`

Reply:
got it. should be fixed in this commit [c33666d](https://github.com/lzell/AIProxySwift/pull/55/commits/c33666d3eb00945859240df2d3797f8d70bb44ff)